Project SF/Views/RoundedNavigationButton.swift

```diff
@@ -24,17 +24,23 @@ struct RoundedNavigationLink<Destination: View>: View {
                     Text(title)
                         .font(.headline)
                         .fontWeight(.semibold)
+                        .padding(12)
+                        .frame(maxWidth: .infinity, minHeight: 55)
+                        .background(Color.blue)
+                        .foregroundColor(.white)
+                        .cornerRadius(16)
+                        .padding()
```

Comment:
Seems like a lot of code duplication, could you move this into a private custom modifier?

Reply:
Gotcha, will fix in a minute